book/src/run_a_node.md

```diff
@@ -0,0 +1,145 @@
+# Run a Node
+
+This document provides detail for users who want to run a Lighthouse beacon node.
+You should be finished with one [Installation](./installation.md) of your choice to continue with the following steps:
+
+1. Perform [Checkpoint sync](#step-1-checkpoint-sync); and
+2. [Connect to an execution engine](#step-2-connect-to-an-execution-engine).
+
+Checkpoint sync is *optional*; however,  we recommend it since it is substantially faster
+than syncing from genesis while still providing all the same features.
+
+## Step 1: Checkpoint sync
```

Comment:
I think we could break this down into more granular steps like:

1. Choose a checkpoint sync provider (describe just how to choose the URL)
2. Set up an execution node (link to EL docs, etc)
3. Run Lighthouse
4. Check logs (example of `INFO Starting checkpoint sync` and `INFO Synced`)
5. Further reading (links to other sections like Becoming a Validator, Validator Monitor, Checkpoint Sync, Merge-Migration, HTTP API, etc).

In step 3 we show an example of the full command-line used for a staking node and a non-staking node respectively, e.g.

Staking:

```bash
lighthouse bn \
  --http \
  --checkpoint-sync-url https://mainnet.checkpoint.sigp.io \
  --execution-endpoint http://localhost:8551 \
  --execution-jwt /secrets/jwt.hex 
```

Non-staking:

```bash
lighthouse bn \
  --checkpoint-sync-url https://mainnet.checkpoint.sigp.io \
  --execution-endpoint http://localhost:8551 \
  --execution-jwt /secrets/jwt.hex \
  --disable-deposit-contract-sync
```